game/src/plugins/skills/mining/src/mining.plugin.kts

```diff
@@ -79,17 +74,11 @@ class ExpiredProspectingAction : DistancedAction<Player> {
     override fun executeAction() {
         mob.sendMessage("There is currently no ore available in this rock.")
     }
-
 }
 
-class ProspectingAction : DistancedAction<Player> {
+class ProspectingAction(val m: Player, val p: Position, val ore: Ore) : DistancedAction<Player>(3 /* PROSPECT PULSES */, true, m, p, 1 /* ORE SIZE */) {
```

Comment:
Can you put the constants (pulses/ore size) into named values? To get constants in Kotlin you use [companion objects:](https://kotlinlang.org/docs/reference/object-declarations.html#companion-objects):

```kotlin
class ProspectingAction ..........

    companion object {
        private val PROSPECT_PULSES = 3
    }
```

Reply:
OK